### ISSUE TITLE
[proposition d'évolution] ajuster la granularité de certaines statistiques

### ISSUE THREAD

**Issue by max5962** [CovidTrackerFr/vitemadose-front#247]
hello,

Je fais suite à cette demande : https://github.com/CovidTrackerFr/vitemadose/issues/365

Les API permet de fournir les informations "1_days, 2_days, 7_days, 28_days et 49_days " il faudra donc les afficher sur le front :)

**Comment by fcamblor** [comment]
Comme évoqué sur #365, les `appointment_schedules` sont plutôt voués à disparaître au profit d'une vision calendaire des prochaines dispos (cf cette PR : #233 )

J'aurais tendance à mettre en standby la problématique évoquée ici, en attendant qu'on y voir plus clair sur le refacto en cours.
Je pense que lorsqu'il sera fait, rajouter une dropdown pour calculer sur les x jours suivants devrait pas être des plus compliqué (ça risque par contre de devoir aller chercher pas mal d'infos : 1 fichier par département multiplié par le nombre de jours dont on souhaite avoir le total complet)

**Comment by max5962** [comment]
Effectivement, cette PR risque de changer pas mal de chose. D'ailleurs beau boulot. ! 
Attendons la suite :)

**Comment by nhumblot** [comment]
Suite à la livraison de #233, le nombre de créneaux disponibles est affiché par jour. Est-ce que des éléments complémentaires sont nécessaires dans le cadre de ce ticket ? J'ai l'impression que l'on a un affichage plus fin que ce qui est demandé.
